test/services/company-officers/companyOfficers.service.test.ts

```diff
@@ -54,21 +54,47 @@ describe('CompanyOfficersService', () => {
 
       const secretary: CompanyOfficer = { ...generateCompanyOfficer(), officerRole: 'secretary' }
       const director: CompanyOfficer = { ...generateCompanyOfficer(), officerRole: OfficerRole.DIRECTOR }
+      const corporateDirector: CompanyOfficer = { ...generateCompanyOfficer(), officerRole: OfficerRole.CORPORATE_DIRECTOR }
+      const corporateNomineeDirector: CompanyOfficer = { ...generateCompanyOfficer(), officerRole: OfficerRole.CORPORATE_NOMINEE_DIRECTOR }
+      const judicialFactor: CompanyOfficer = { ...generateCompanyOfficer(), officerRole: OfficerRole.JUDICIAL_FACTOR }
       const manager: CompanyOfficer = { ...generateCompanyOfficer(), officerRole: 'cicmanager' }
```

Comment:
is that a valid role? if so, can we introduce enum for this one?

Reply:
This is not in the API

Reply:
It is a role but it isn't valid for closing a company, so we don't actually use it outside of testing